app/Models/Task.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    use HasFactory;
```

Comment:
good to write the table name here. I know that is not required in this case, but a good practice to avoid typos or snake case issues for muti worded models.
```
protected $table = 'tasks';
```

Reply:
ok